program/src/state/frequency.rs

```diff
@@ -1,8 +1,9 @@
 use borsh::{BorshDeserialize, BorshSerialize};
+use shank::ShankAccount;
 
 use super::{Key, SolanaAccount};
 
-#[derive(BorshSerialize, BorshDeserialize, PartialEq, Eq, Debug, Clone)]
+#[derive(BorshSerialize, BorshDeserialize, PartialEq, Eq, Debug, Clone, ShankAccount)]
```

Comment:
Looks like we need to add the `ShankAccount` so that stuff is generated saying this will be an Account that holds data of this struct type.  Just curious did you actually need it as part of this CLI PR or just added it because we will need it in general soon, as we implement the Frequency PDA handling?

Reply:
Solita was failing to generate the SDK because it couldn't find the accounts.ts file. We're in weird territory because technically there should be a RuleSet.ts file as well but it can't be a shank account.